main/inc/lib/main_api.lib.php

```diff
@@ -355,6 +355,13 @@
 define('CATALOG_SESSIONS', 1);
 define('CATALOG_COURSES_SESSIONS', 2);
 
+//Hook
```

Comment:
Add comment to describe constants
Update constants name to:

HOOK_EVENT_TYPE_PRE
HOOK_EVENT_TYPE_POST
HOOK_EVENT_TYPE_ALL